.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - critical
```

Comment:
Should we add an other label like `no stale` to keep specific issues open?

Reply:
Yes, or `wont stale` (to be consistent with `wont fix`).
Let me know and I'll be pushing again.

Reply:
Or something like `keep open` or just `keep`

Reply:
Done!